whipper/command/cd.py

```diff
@@ -290,6 +291,14 @@ def add_arguments(self):
                                  help="whether to continue ripping if "
                                  "the disc is a CD-R",
                                  default=False)
+        self.parser.add_argument('-C', '--cover-art',
+                                 action="store", dest="fetch_cover_art",
+                                 help="Fetch cover art and save it as "
```

Comment:
```suggestion
                                 help="fetch cover art and save it as "
```
I just noticed a small detail that this shouldn't be capital just to be consistent 😉.